scripts/build-data/file-contributors.ts

```diff
@@ -24,16 +24,21 @@ export default {
 const getAllGHCommits = async (task: any, page = 1) => {
   try {
     task.output = `Getting commits: ${page * 100 - 99} - ${page * 100}`;
-    const request = await fetch(url.format({
-      protocol: 'https',
-      hostname: 'api.github.com',
-      pathname: 'repos/ionic-team/ionic-docs/commits',
-      query: {
-        access_token: process.env.GITHUB_TOKEN,
-        per_page: 100,
-        page
+    const request = await fetch(
+      url.format({
+        protocol: 'https',
+        hostname: 'api.github.com',
+        pathname: 'repos/ionic-team/ionic-docs/commits',
+        query: {
+          per_page: 100,
+          page
+        }
+      }), {
+        headers: {
+          'Authorization': process.env.GITHUB_TOKEN !== undefined ? process.env.GITHUB_TOKEN : ''
```

Comment:
According to the [blog post](https://developer.github.com/changes/2020-02-14-deprecating-password-auth/) you need `token` in front of the token:
```suggestion
          'Authorization': process.env.GITHUB_TOKEN !== undefined ? `token ${process.env.GITHUB_TOKEN}` : ''
```